packages/flutter/test/material/text_field_test.dart

```diff
@@ -8919,6 +8919,1124 @@ void main() {
     },
   );
 
+  group('Triple tap/click', () {
```

Comment:
It's a bummer that we have to duplicate all of these tests for Cupertino and Material, but I guess they won't work on EditableText because of the code changes in text_selection.dart.

Someday we need to make it easier to share these kinds of tests!

Reply:
Agreed, it would definitely help with velocity if we only had to write these tests once.